src/include/simeng/arch/aarch64/helpers/sve.hh

```diff
@@ -1631,6 +1634,44 @@ class sveHelp {
     }
     return {out, 256};
   }
+
+  /** Helper function for SVE instructions store instructions.
+   * T represents the type of operands (e.g. for zn.d, T = uint64_t).
+   * This function merges store data for contiguous active elements, and return
```

Comment:
returns a vector containing store data for all merged contiguous active elements.

Reply:
Additional to this, the comment should be formatted to match all other helper functions.